### ISSUE TITLE
Trade macro Won't open on windows 10

### ISSUE THREAD

**Issue by Nakimushi** [PoE-TradeMacro/POE-TradeMacro#763]
_TEMPLATE_ (Remove the explanations here from your post) 

Make sure:
* That you have checked the Wiki/FAQ before creating any issue.
* That poe.trade/GGG's website don't have any issues if loading the script takes too long or failes.

Information to include:
* AHK (including 32/64bit) version if you have any kind of script errors.
* Windows version could be relevant.

Fill out everything that may be relevant and remove everything except these informations below:  

AHK version: 1.29
Windows version: 10

Basically when I try to opne the script it pops up and asks which program I would like to open with instead of it just going into my hotbar. Can't figure out why it worked fine in the past

**Comment by aRTy42** [comment]
Go to your script folder, right click the Run_TradeMacro.ahk and select properties. In the first tab "General" it should say "Opens with:" almost at the top. The selection should be your autohotkey installation, in my case "AutoHotkey Unicode 64-bit". Fix that selection.

**Comment by Nakimushi** [comment]
That worked thanks
